apps/api/src/server/utils/mapping/map-application.js

```diff
@@ -4,13 +4,11 @@ import { mapKeysUsingObject } from '../../utils/mapping/map-keys-using-object.js
 import { mapValuesUsingObject } from '../../utils/mapping/map-values-using-object.js';
 
 /**
- *
- * @typedef {{id: number, modifiedDate: number, reference: string}} ApplicationResponse
+ * @typedef {{id: number, reference: string, title:string, description: string, status: string, modifiedDate: number, publishedDate?}} ApplicationResponse
  */
 
 /**
- *
- * @param {import('@pins/api').Schema.Application} application
+ * @param {import('@pins/api').Schema.Application[]} application
```

Comment:
@MariaValeroGonzalezKC I forgot to undo the extra [] I added here, line should read:

* @param {import('@pins/api').Schema.Application} application

not 

* @param {import('@pins/api').Schema.Application[]} application